chart/templates/generated/role.yaml

```diff
@@ -10,7 +10,6 @@ rules:
     verbs:
       - create
       - delete
-      - get
```

Comment:
Hm there's a chance this `get` being removed from the generated file was in fact manually added previously for a reason, we might want to add it to rules defined through kubebuilder to keep it

Reply:
I think it's not relevant due to how informer works
Informer never does get, it does list+watch
all the calls are referenced here, none of the ones done through chaos-controller are using a standard k8s client

TBH, I changed this because we lacked a get (for the services) and it worked without it

I will try without on staging and see how it goes

Reply:
In the end I took the approach you suggested and [added](https://github.com/DataDog/chaos-controller/pull/734/commits/90ad0edabb1deb89421d3998ebb6bc3156cb3720) the permissions as before 😄 
I don't want to retest all disruptions again and will fix this at a later stage as it's not harming in any way today.